src/mpv/events.rs

```diff
@@ -71,31 +71,35 @@ impl Mpv {
     }
 }
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Clone, Debug)]
```

Comment:
Note: I had to remove `PartialEq`. It is unclear to me how to check partial equality for two `PropertyData::MpvNode`s.

Reply:
Note: this is a breaking change.